activities/supporting-codebase-governance/exercise/index.md

```diff
@@ -0,0 +1,24 @@
+---
+type: index
+redirect_from:
+    - activities/workshops/governance-exercise
```

Comment:
Isn't moving this down in the nested tree siloing?

Reply:
With my [later comment](https://github.com/publiccodenet/about/pull/649#issuecomment-642547808), is this resolved?

Reply:
So the answer is yes, but in this case it is okay?

Reply:
it's not changing the depth, it is however changing where this is located. In stead of this being located under 'Workshops', it's under 'Supporting governance' as this workshop is (correct me if I'm wrong) part of that activity.

I see it like this:

* Workshops: documentation of practice around workshops.
* Supporting codebase governance: documentation of practice around supporting codebase stewardship.

My thinking was that these workshops are something everyone in 'supporting codebase governance' works on whilst only a few people working on 'workshops' will use it, since that also has the people doing other workshops in it.

See also: https://github.com/publiccodenet/about/pull/649#issuecomment-642547808